ansible/roles/controller/tasks/deploy.yml

```diff
@@ -15,7 +15,7 @@
     mode: 0777
   become: true
 
-- name: (re)start controller
+- name: (re)start controller "{{ db.whisk.actions }}"
```

Comment:
yes - will remove; will need to resolve the db prefix without consul; we were getting away with this undetected.